io.emeric.watchflower.yml

```diff
@@ -0,0 +1,98 @@
+app-id: io.emeric.watchflower
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: watchflower
+
+rename-desktop-file: watchflower.desktop
+rename-appdata-file: watchflower.appdata.xml
+#rename-icon: watchflower
```

Comment:
I'm under the impression you'll need this, from a quick glance at the source.

Reply:
Ok so "rename-icon: watchflower.svg" ? The SVG file is next to the desktop and appdata file already.

Reply:
just `rename-icon: watchflower`

Reply:
thanks, I'll push a release tomorrow and update this pull request.